specifyweb/frontend/js_src/lib/components/Forms/Save.tsx

```diff
@@ -195,7 +201,10 @@ export function SaveButton<SCHEMA extends AnySchema = AnySchema>({
   const copyButton = (
     label: LocalizedString,
     description: LocalizedString,
-    handleClick: () => Promise<SpecifyResource<SCHEMA>>
+    handleClick: () =>
+      | Promise<RA<SpecifyResource<SCHEMA>>>
+      | Promise<SpecifyResource<SCHEMA>>,
```

Comment:
```suggestion
    handleClick: () => Promise<RA<SpecifyResource<SCHEMA>> | SpecifyResource<SCHEMA>>
```

Reply:
(optional) but it would be even better if you change the type to just always be an array (like we did in case of handleAdd() in other places)
that would simplify the type-checking. the consumer code can then check if array length is 1 or not